tests/test_map_mode.c

```diff
@@ -0,0 +1,82 @@
+#include "map/mode.h"
+#include "test_define.h"
+
+#define chid "UANFAVTSAXZMYUWRECNAOJDAQVTTORVGJCCISMZYAFFU9EYLBMZKEJ9VNXVFFGUTCHONEYVWVUTBTDJLO"
+#define chid1 "ONMTPDICUWBGEGODWKGBGMLNAZFXNHCJITSSTBTGMXCXBXJFBPOPXFPOJTXKOOSAJOZAYANZZBFKYHJ9N"
```

Comment:
I have no idea about the naming `chid1`. Why?

Reply:
`chid1` is the next channel that announced, it is used commonly under mam. On second thought, it may not be a good abstraction. I'll rename it to `new_chid` .